staging/src/k8s.io/api/networking/v1/types.go

```diff
@@ -49,7 +49,11 @@ const (
 	PolicyTypeEgress PolicyType = "Egress"
 )
 
-// NetworkPolicySpec provides the specification of a NetworkPolicy
+// NetworkPolicySpec provides the specification of a NetworkPolicy.  A suite of network
+// policies allow traffic ONLY if BOTH the egress AND the ingress are allowed.  For example,
+// if ingress is allowed by egress is not, or egress is allowed by ingress is not, the traffic
```

Comment:
same comment as earlier:

if ingress is allowed **but** egress is not, or egress is allowed **but** ingress is not

Reply:
👍